packages/insomnia-cookies/src/cookies.ts

```diff
@@ -43,18 +39,21 @@ module.exports.jarFromCookies = function(cookies) {
   return jar;
 };
 
-module.exports.cookieToString = function(cookie) {
+export const cookieToString = (cookie: Parameters<typeof Cookie.fromJSON>[0] | Cookie) => {
   // Cookie can either be a plain JS object or Cookie instance
   if (!(cookie instanceof Cookie)) {
-    cookie = Cookie.fromJSON(cookie);
-  }
+    cookie = Cookie.fromJSON(cookie) as Cookie;
 
+    if (cookie === null) {
+      throw new Error('asdf');
```

Comment:
Nice catch! I guess this would've just thrown down on `let str = cookie.toString();` previously?
Probably worth changing the error message however 😅

Reply:
good catch, yes, another thing that typescript forces us to handle that was not handled before.

I changed it to
```
      throw new Error(`Unable to read cookie: ${cookie}`);
```